Sources/Views/MessageInputBar.swift

```diff
@@ -421,7 +421,7 @@ open class MessageInputBar: UIView {
     open func calculateIntrinsicContentSize() -> CGSize {
         
```

Comment:
What about the `left` and `right` stack views? Their height can affect the total size of the input bar if their height is greater than the `inputTextView.height` & `topStackView.height`.

Reply:
I think `left` or `right`'s `height` would not greater than that based on constraints.